Gems/Atom/Feature/Common/Editor/Scripts/ColorGrading/lut_helper.py

```diff
@@ -0,0 +1,233 @@
+
+import ColorGrading.initialize
+ColorGrading.initialize.start()
+
+try:
+    import OpenImageIO as oiio
+    pass
+except ImportError as e:
+    _LOGGER.error(f"invalid import: {e}")
+    sys.exit(1)
+# ------------------------------------------------------------------------
+
+
+# ------------------------------------------------------------------------
+# Transform from high dynamic range to normalized
+def ShaperInv(bias, scale, v):
+    return math.pow(2.0, (v - bias)/scale)
+
+# Transform from normalized range to high dynamic range
+def Shaper(bias, scale, v):
+    
+    if v > 0.0:
+        value = math.log(v, 2.0) * scale + bias
+    
+    elif v <= 0.0:
+        value = math.log(0.002, 2.0) * scale + bias
+        _LOGGER.debug(f'Clipping a value: bias={bias}, scale={scale}, v={v}')
+
+    return value
+
+def GetUvCoord(size, r, g, b):
+    u = g * lutSize + r
+    v = b
+    return (u, v)
+
+def clamp(v):
+    return max(0.0, min(1.0, v))
+
+shaperPresets = {"Log2-48nits":(-6.5, 6.5),
+                 "Log2-1000nits":(-12.0, 10.0),
+                 "Log2-2000nits":(-12.0, 11.0),
+                 "Log2-4000nits":(-12.0, 12.0)}
+
+operations = {"pre-grading":0, "post-grading":1}
+
+parser = argparse.ArgumentParser()
+parser.add_argument('--i', type=str, required=True, help='input file')
+parser.add_argument('--shaper', type=str, required=True, help='shaper preset. Should be one of \'Log2-48nits\', \'Log2-1000nits\', \'Log2-2000nits\', \'Log2-4000nits\'')
+parser.add_argument('--op', type=str, required=True, help='operation. Should be \'pre-grading\' or \'post-grading\'')
+parser.add_argument('--o', type=str, required=True, help='output file')
+parser.add_argument('-l', dest='write3dl', action='store_true', help='output 3dl file')
+parser.add_argument('-a', dest='writeAsset', action='store_true', help='write out azasset file')
+args = parser.parse_args()
+
+# Check for valid shaper type
+invalidShaper = (0,0)
+invalidOp = -1
+
+shaperLimits = shaperPresets.get(args.shaper, invalidShaper)
+if shaperLimits == invalidShaper:
+    _LOGGER.error("invalid shaper")
+    sys.exit(1)
+op = operations.get(args.op, invalidOp)
+if op == invalidOp:
+    _LOGGER.error("invalid operation")
+    sys.exit(1)
+
+# input validation
+from pathlib import Path
+input_file = Path(args.i)
+if input_file.is_file():
+    # file exists
+    pass
+else:
+    FILE_ERROR_MSG = f'File does not exist: {input_file}'
+    _LOGGER.error(FILE_ERROR_MSG)
+    #raise FileNotFoundError(FILE_ERROR_MSG)
+    sys.exit(1)
+
+# Read input image
+#buf = oiio.ImageBuf("linear_lut.exr")
+buf = oiio.ImageBuf(args.i)
+inSpec = buf.spec()
+_LOGGER.info(f"Resolution is x:{inSpec.height}, y:{inSpec.width}")
+
+if inSpec.height < 16:
+    _LOGGER.info(f"invalid input file dimensions: x is {inSpec.height}. Expected LUT with height dimension >= 16 pixels")
+    sys.exit(1)
+    
+def log2(x):
+    return (math.log10(x) /
+            math.log10(2))
```

Comment:
You could use math.log(x, 2.0) here, then replace calls below to that function to just log2(x)